api/pkg/cli/formatter/field.go

```diff
@@ -108,6 +108,7 @@ func breakString(desc string, width, titleLength int) string {
 	for i := firstLineEnd; i < descLength; i = i + spaceIndex {
 		if descLength < i+width {
 			sb.WriteString(desc[i:])
+			break
```

Comment:
Why we need this? the condition in for look will do this

Reply:
`i` is incremented by adding `spaceIndex` but in the first case we didn't need to calculate it as it is the last line so it adds `0` in `i` and it goes in an infinite loop, in else we calculate `spaceIndex` as we need to break the line at that index.